delivery_carrier_required_package_measurement/wizard/choose_delivery_package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class ChooseDeliveryPackage(models.TransientModel):
+    _inherit = "choose.delivery.package"
+
+    pick_name = fields.Char(related="picking_id.name")
+
+    def _default_dimension_uom_id(self):
+        val = self.env["product.template"]._get_length_uom_id_from_ir_config_parameter()
+        return val
+
+    package_height = fields.Integer()
+    package_length = fields.Integer()
+    package_width = fields.Integer()
+    dimension_uom_id = fields.Many2one(
+        "uom.uom",
+        default=_default_dimension_uom_id,
```

Comment:
```suggestion
        default=lambda self: self._default_dimension_uom_id(),
```
Then you can move the method below.

Reply:
Changed